packages/suite/src/components/suite/TrafficLightOffset.tsx

```diff
@@ -0,0 +1,39 @@
+import { isDesktop, isMacOs } from '@trezor/env-utils';
+import React from 'react';
+import styled from 'styled-components';
+
+type Props = {
+    children?: React.ReactNode;
+    offset?: number;
+};
+
+// this element fixes not being able to drag window: https://github.com/electron/electron/issues/5678
+const WindowAppHandler = styled.div`
```

Comment:
```suggestion
// See: https://github.com/electron/electron/issues/5678
const FixForNotBeingAbleToDragWindow = styled.div`
```

Better to incorporate the information into naming rather then comments as they tend to decay/being not-updated.

Also this is then clear at place where this is used.